sdk/appconfiguration/app-configuration/src/index.ts

```diff
@@ -118,18 +164,69 @@ export class AppConfigurationClient {
     return this.client.deleteConfigurationSetting(key, options);
   }
 
+  /**
+   * Gets a setting from the Azure App Configuration service.
+   * 
+   * Example code:
+   * ```ts
+   * const setting = await client.getConfigurationSetting(key, { label: "MyLabel" });
+   * ```
+   * @param key The name of the key.
+   * @param options Optional parameters for the value being retrieved.
+   */
   getConfigurationSetting(key: string, options?: GetConfigurationSettingOptions): Promise<GetConfigurationSettingResponse> {
     return this.client.getConfigurationSetting(key, options);
   }
 
+  /**
+   * Lists settings from the Azure App Configuration service, optionally filtered by label, 
+   * accept date time or name.
+   * 
+   * Example code:
+   * ```ts
+   * // find all settings with label "MyLabel"
+   * const matchingSettings = await client.listConfigurationSettings({ label: "MyLabel" });
+   * ```
+   * @param options Optional parameters for the settings being listed.
+   */
   listConfigurationSettings(options?: ListConfigurationSettingsOptions): Promise<ListConfigurationSettingsResponse> {
     return this.client.listConfigurationSettings(options);
   }
 
+  /**
+   * Lists revisions of a set of keys within the Azure App Configuration service.
+   * 
+   * Example code:
+   * ```ts
+   * // create a new setting
+   * await client.addConfigurationSetting(key, { value: "foo1" });
+   * 
+   * // update it with a new value
+   * await client.setConfigurationSetting(key, { value: "foo2" });
+   * 
+   * // retrieves all revisions matching that label
+   * const result = await client.listRevisions({ key: [key] });
```

Comment:
The samples are mainly for the usage of the method in question. I wonder if prior setup is needed here and that the stand alone files would be a better place for this.
@bterlson thoughts?

Reply:
I removed it for now and tried to make it a bit more obvious by changin the 'result' return value to have a name (revisionsForMyKey).